go.mod

```diff
@@ -7,22 +7,23 @@ require (
 	github.com/gofrs/flock v0.8.1
 	github.com/golang/protobuf v1.5.3
 	github.com/google/uuid v1.6.0
-	github.com/gorilla/handlers v1.5.1
-	github.com/gorilla/mux v1.8.0
-	github.com/longhorn/backupstore v0.0.0-20231114103026-af339bb498d3
+	github.com/gorilla/handlers v1.5.2
+	github.com/gorilla/mux v1.8.1
+	github.com/longhorn/backupstore v0.0.0-20240219094812-3a87ee02df77
+	github.com/longhorn/go-common-libs v0.0.0-20240109042507-23627e6416b7
```

Comment:
Should it be updated to the latest commit id [e7176c3321562849926993bb35e63a6beb877ec4](https://github.com/longhorn/go-common-libs/commits/main/)?

Reply:
that should be fine, as bot automatically updates this. We can wait for the next update.